README.md

```diff
@@ -19,7 +19,7 @@ let uri = "127.0.0.1:9042";
 
 let session: Session = SessionBuilder::new().known_node(uri).build().await?;
 
-let result = session.query("SELECT a, b, c FROM ks.t", &[]).await?;
+let result = session.query_unpaged("SELECT a, b, c FROM ks.t", &[]).await?;
```

Comment:
Well, this is precisely the kind of query and place you want people NOT to use `query_unpaged`. :-)

Reply:
...which explains why we need this refactor that much :P